bowtie/templates/report.html.j2

```diff
@@ -1,6 +1,54 @@
 <!DOCTYPE html>
 <html lang="en">
 
+<style>
+  .details-required {
```

Comment:
This might be a merge mistake -- the style tag belongs in the head tag if we're going to have it -- though a few more things below look like they should be doable with normal bootstrap classes -- but can you move all this into the style tag in the head tag?

Reply:
and Done!
I cannot express my gratitude enough for your incredible patience throughout this journey. Your unwavering support and guidance have been invaluable to me. Your help in identifying and correcting my mistakes has been particularly meaningful. Thank you so much for everything.

Reply:
No worries at all, definitely not a race! Thanks for the efforts!